examples/jsm/exporters/USDZExporter.js

```diff
@@ -375,7 +480,26 @@ function buildMaterial( material ) {
 
 	if ( material.metalnessMap !== null ) {
 
-		parameters.push( `${ pad }float inputs:metallic.connect = </Textures/Texture_${ material.metalnessMap.id }.outputs:b>` );
+		parameters.push( `${ pad }float inputs:metallic.connect = </Materials/Material_${ material.id }/Texture_${ material.metalnessMap.id }_metalness.outputs:b>` );
+
+		texture = material.metalnessMap;
+		//texturesTransforms.push(prepareTextureTransform(texture));
```

Comment:
Metalness and Roughness maps are often the same map but uses different channels so to avoid duplicated defs for same texture id just use the same as metalness. For now it supposes that if we have roughness map there should also be e matalness one.